FinniversKit/Sources/Components/ChristmasPromotionView/ChristmasPromotionView.swift

```diff
@@ -144,4 +158,8 @@ private extension UIColor {
     static var shadowColor: UIColor {
         return UIColor(hex: "475569")
     }
+    
+    static var bgColor: UIColor {
+        return .dynamicColorIfAvailable(defaultColor: .milk, darkModeColor: .blueGray700)
```

Comment:
Android added this color as "bgTile" (or something similar), I think we should consider that too since we are now using it many places. But that we can change in another PR later 👍 Or wait until the ongoing project to clear up the "mess" with new colors in the web design system is ready 😛